Makefile

```diff
@@ -1,10 +1,49 @@
-deps-update:
+# Go parameters
+ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+GOCMD=go
```

Comment:
I think this Makefile may be overly complex for bond cni - is this based on template?

Reply:
yes this I was looking at the makefile for the sr-iov plugin. I can trim this stuff out to make it more simple